pkg/client/unversioned/extensions.go

```diff
@@ -130,24 +130,24 @@ func NewExtensionsOrDie(c *Config) *ExtensionsClient {
 
 func setExtensionsDefaults(config *Config) error {
 	// if experimental group is not registered, return an error
-	g, err := latest.Group("extensions")
-	if err != nil {
-		return err
+	if !extlatest.IsEnabled() {
```

Comment:
Apart from what liggitt said, another reason to use the central latest is to minimize the import graph.

Sorry I didn't catch up with this PR earlier. Could you hold on for a day? I want to discuss it with @lavalamp.